README.md

```diff
@@ -9,19 +9,77 @@ helm repo add kuadrant https://kuadrant.io/helm-charts/ --force-update
 
 See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation.
 
-## Installing a Chart
+## Kuadrant operator chart
+The official Kuadrant Kubernetes Operator
 
-Installing a Kuadrant Chart, i.e: `limitador-operator`
-```shell
-helm install limitador-operator kuadrant/limitador-operator
+### Install Dependencies
+Follow the steps below to install the following dependencies for Kuadrant Operator:
+- [Gateway API](https://gateway-api.sigs.k8s.io/)
+- [cert-manager](https://cert-manager.io/)
+- A Gateway Provider [Istio](https://istio.io/latest/docs/ambient/install/helm/) or [Envoy Gateway](https://gateway.envoyproxy.io/)
+
+#### Kubernetes Gateway API:
+
+ ```sh
+ kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.1.0/standard-install.yaml
+ ```
+
+#### cert-manager
+
+ ```sh
+ helm repo add jetstack https://charts.jetstack.io --force-update
+ helm install \
+   cert-manager jetstack/cert-manager \
+   --namespace cert-manager \
+   --create-namespace \
+   --version v1.15.3 \
+   --set crds.enabled=true
+ ```
+
+#### Gateway Provider (I.E. Istio):
```

Comment:
Nice having this inline.
We could also link out to https://gateway.envoyproxy.io/latest/install/install-helm/ for users that want to try EG.